### DISCUSSION TITLE
"x" should not show when not rights are given to delete an M2M item

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12612] [Feature Requests]
Having an M2M field on a table `Master`  related to a table `Related` with `Master_Related` No Access to Delete 
should not show the `X` to delete the M2M field
Instead of
![x](https://user-images.githubusercontent.com/13746133/162152060-f8f57ba6-0cb0-4b01-81be-e2bce8512cb8.png)
 we should have
![x-](https://user-images.githubusercontent.com/13746133/162152119-4cfe9a2a-fb78-451c-bdf3-5e708d94703c.png)

_After Saving, the item is not deleted which is correct.
But we should not let the user POV to think he could._

--

In the same vein, 
when `No Access` is given, even to See any M2M items on `Master_Related`, the name of the M2M should not appear at all.

_The name of the field still shows (with nothing under)
and from the user's POV it looks wrongly that something is missing._

**Comment by rijkvanzanten** [comment]
> No Access to Delete

That's interesting though, cause the `x` isn't to _delete_ the "related" item, but it's to remove the junction row that relates the current item to the related item 🤔 Or do you mean the delete permissions on the junction table?

**Reply by Pachat** [reply]
Right: I mean the delete permissions on the junction table `Master_Related`
and in the second case, no permissions on the junction table `Master_Related`

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
